pkgs/by-name/ba/basilk/package.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "basilk";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "gabalpha";
+    repo = pname;
+    rev = "refs/tags/${version}";
+    sha256 = "sha256-b77vVf+WbDNzKwGaMJvgMEMCC41h5TXmg45OM9g4v+4=";
+  };
+
+  cargoHash = "sha256-ZYyRUqWbwAoNaGuCXJvqWTgUm5jNpihqvPe5SsfjEq4=";
+
+  meta = with lib; {
```

```suggestion
  meta =  {
```

---

```suggestion
  meta = {
```
Remove the with lib as it is only reference in two places within meta